### ISSUE TITLE
build check failure (read.met)

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1368]
A winbuilder test, i.e.
```R
devtools::build_win(version="R-release")
```
on R-release gives as follows. I suspect it's an encoding problem in `read.met`, since the test works fine on my machine. The full test is at https://win-builder.r-project.org/h8ZvCvD8adgm/00check.log but that site will only live for 48 hours or whatever the default is.
```
Running the tests in 'tests/testthat.R' failed.
Last 13 lines of output:
  Loading required package: testthat
  > 
  > test_check("oce")
  -- 1. Failure: (unknown) (@test_met.R#3)  --------------------------------------
  names(d@data) not equal to c(...).
  1/15 mismatches
  x[3]: "Dew.Point.Temp..8C."
  y[3]: "dewPoint"
  
  == testthat results  ===========================================================
  OK: 1066 SKIPPED: 35 FAILED: 1
  1. Failure: (unknown) (@test_met.R#3) 
  
  Error: testthat unit tests failed
  Execution halted
** running tests for arch 'x64' ... [26s] ERROR
  Running 'testthat.R' [25s]
Running the tests in 'tests/testthat.R' failed.
Last 13 lines of output:
  Loading required package: testthat
  > 
  > test_check("oce")
  -- 1. Failure: (unknown) (@test_met.R#3)  --------------------------------------
  names(d@data) not equal to c(...).
  1/15 mismatches
  x[3]: "Dew.Point.Temp..8C."
  y[3]: "dewPoint"
  
  == testthat results  ===========================================================
  OK: 1066 SKIPPED: 35 FAILED: 1
  1. Failure: (unknown) (@test_met.R#3) 
  
  Error: testthat unit tests failed
  Execution halted
```

**Comment by dankelley** [comment]
The offending char is shown in the screenshot. On my machine, this turns into a `.` but on the winbuilder machine (in Europe, I think) it is `8`.  (I will add more notes to this comment as I work on this, to avoid email clutter to other devels.)
<img width="293" alt="screen shot 2018-01-21 at 8 41 08 am" src="https://user-images.githubusercontent.com/99469/35194266-05c4e5c2-fe87-11e7-8d81-30e3b89369eb.png">

**Comment by dankelley** [comment]
Fixed in "develop" commit aa308373a60dd8a5fc67bf00be00f76eb9eb9248 by modifying the pertinent regexp. (My solution was to use a `.*` whereas before I had a `...` in a regexp; I think this will be safe because I also have some pretty specific char matchers up to `$`, so I don't expect false positives.)

I use env-can data a couple of times a year, so likely I'll be able to notice new problems. My plan will be to make any new code continue to work with old data files, **unless** env-can changes the files in radical ways that I cannot detect easily. If they ever release an API, I'll see if I can accommodate it.
